### PR TITLE
Get mapviewer working again

### LINKED ISSUE #5: enable-mapping is not compatible with enable 4.6.0
After updating enable to 4.6.0, one of our packages using enable-mapping is failing; there seems to be an infinite recursion going on between the `ViewPort` and the `Canvas`. My suspicion is that recent changes to enable's `ViewPort` need to be compensated for in the enable-mapping version.

### PR DESCRIPTION
**Pull request by senganal**
Closes #5 

This unambitious PR refactors the `MappingViewport` to get the map viewer in canopygeo functional. Nothing more..

### CODE REVIEW

File: mapping/enable/viewport.py [hunk 1]
```diff
@@ -48,18 +52,38 @@ def _get_longitude(self):
     def _set_longitude(self, val):
         self.geoposition[1] = val 
 
-    @on_trait_change('geoposition, bounds, bounds_items')
-    def _update_position(self, object, name, old, new):
-        # Update position
+    def _component_bounds_updated(self, old, new):
+        super(MappingViewport, self)._component_bounds_updated(old, new)
+        # FIXME: This seems to be needed to make sure the zooming works
```
Comment (mdickinson) [inline mapping/enable/viewport.py:57]: Please could you open issues for these #FIXME comments?

**Review by mdickinson — APPROVED** [review]
LGTM. Please could we have issues open for the FIXME comments? (Ideally, with links to those issues from the code comments.)

File: mapping/enable/viewport.py (hunk 1 above)
Comment (senganal) [reply mapping/enable/viewport.py:57]: done

**Comment by mdickinson** [comment]
LGTM. Thanks! As soon as this is merged I'll update the canopy-data dependencies so that we can get enthought/canopy_data#558 passing.

**Comment by senganal** [comment]
Thanks!

**Pull request merged**